<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppletTestAndDebugUi</class>
 <widget class="QWidget" name="AppletTestAndDebugUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>442</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Test And Debug</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="StyledDlgItemsBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="gotoWebsiteButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>visit http://www.nolimitconnect.com</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="m_ShowAppInfoButton">
          <property name="text">
           <string>Show App Info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_ShowLogButton">
          <property name="text">
           <string>Show Log</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_BrowseFilesButton">
          <property name="text">
           <string>Browse Files</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="m_MyUrlLabel">
          <property name="text">
           <string>My URL:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_MyUrlEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_CopyMyUrlButton">
          <property name="text">
           <string>Copy To Clipboard</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="m_TestUrlsComboBox"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="m_TestUrlLabel">
          <property name="text">
           <string>Remote URL:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_TestUrlEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="m_CopyTestUrlButton">
          <property name="text">
           <string>Copy To Clipboard</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="m_IsMyPortOpenButton">
          <property name="text">
           <string>Test Is My Port Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_QueryHostIdButton">
          <property name="text">
           <string>Query Host ID </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="m_GenerateGuidButton">
          <property name="text">
           <string>Generate A GUID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_PurgeCacheButton">
          <property name="text">
           <string>Purge Cache</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="m_HostClientTestButton">
          <property name="text">
           <string>Test Hosts As Client</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_HostServiceTestButton">
          <property name="text">
           <string>Test Host Service</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="m_ListActionButton">
          <property name="text">
           <string>Log checked lists</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_AnnouncedCheckBox">
          <property name="text">
           <string>Announced</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_HostJoinedCheckBox">
          <property name="text">
           <string>Host Joined</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_ClientJoinedCheckBox">
          <property name="text">
           <string>Client Joined</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="m_CopyResultToClipboardButton">
        <property name="text">
         <string>Copy Results To Clipboard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="m_InfoPlainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
